docs/source/api/word-functions.rst

```diff
@@ -0,0 +1,41 @@
+.. Copyright (c) 2022, M. T. Whyte
+
+   Distributed under the terms of the GPL license version 3.
+
+   The full license is in the file LICENSE, distributed with this software.
+
+   This file only exists because Breathe always displays all members when
+   documenting a namespace, and this is nicer for now.
+
+.. cpp:namespace:: libsemigroups
+
+Word functions
+--------
+
+This page contains the documentation for functions which may be useful for manipulating words and strings.
+
+Contents
+~~~~~~~~
+
+.. cpp:namespace:: libsemigroups::fpsemigroup
+
+.. list-table::
+   :widths: 50 50
+   :header-rows: 0
+
+   * - :cpp:any:`operator<<`
```

Comment:
I think you'll need to add the arguments to this to make the doc work, there should be plenty of similar examples in the doc already.